php/EE/Model/Site.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace EE\Model;
+
+/**
+ * Site model class.
+ */
+class Site extends Base {
+
+	protected static $table = 'sites';
```

Comment:
Add variable doc/comment. And check other places in this PR and add the comment to respective places.